api/migrations/0002_news.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
@PetrDlouhy, yes, we need to. It's already running on staging and live database.